packages/data-types/src/command.ts

```diff
@@ -0,0 +1,165 @@
+import yargs from 'yargs';
+import path from 'path';
+import fs from 'fs';
+import { debugLogger, get } from '@terascope/utils';
+import { DataType, DataTypeConfig } from './index';
+
+// @ts-ignore
+const logger = debugLogger('data-type-cli');
+// change pathing due to /dist/src issues
+const packagePath = path.join(__dirname, '../../package.json');
+const { version } = JSON.parse(fs.readFileSync(packagePath, 'utf8'));
+
+yargs
+    .command('es-mapping', 'create an elasticsearch mapping from the provided data type',
+    {
+        name: {
+            alias: 'n',
+            describe: 'name of the type that the record has for elasticsearch mappings',
+            demandOption: true
+        },
+        settings: {
+            alias: 's',
+            describe: 'additional settings added to the mapping'
+        }
+    },
+     wrapper(getESMapping)
+    )
+    .command('gql', 'creates an graphql schema for the data type',
+    {
+        name: {
+            alias: 'n',
+            describe: 'name of the type for the generated schema',
+            demandOption: true
+        }
+    },
+    wrapper(getGraphQlSchema)
+    )
+    .command('xlucene', 'create an elasticsearch mapping from the provided data type',
+        {},
+        wrapper(getXluceneValues)
+    )
+    .example('yarn data-types es-mapping --name=event -s index.number_of_shards:20 -s index.number_of_replicas:1', '')
+    .example('\nyarn data-types gql --name=event', '')
```

Comment:
Same as above, also there is no reason to add a newline in front of this.

```suggestion
    .example('$0 gql --name=event', '')
```

Reply:
fixed